wdl/transforms/new-base/src/main/scala/wdl/transforms/base/wdlom2wom/CommandPartElementToWomCommandPart.scala

```diff
@@ -57,8 +57,9 @@ case class WdlomWomStringCommandPart(stringCommandPartElement: StringCommandPart
 
 case class WdlomWomPlaceholderCommandPart(attributes: PlaceholderAttributeSet, expression: WdlomWomExpression)
     extends CommandPart {
-  def attributesToString: String = attributes.toWdlV1
+  private def attributesToString: String = attributes.toWdlV1
   // Yes, it's sad that we need to put ${} here, but otherwise we won't cache hit from draft-2 command sections
+  // Re-writes e.g. `echo $((5 + ~{x.a}))` commands to `echo $((5 + ${x.a}))` so they evaluate to equal
   override def toString: String = "${" + s"$attributesToString${expression.expressionElement.toWdlV1}" + "}"
```

Comment:
I'm not sure this is worth it or if I even agree with the original decision, but adjudicating that seemed out of scope for this PR. It does mean we have to retain the ability to write any expression element.